Automattic/wp-calypso
# [WIP] DO NOT MERGE Anchor: Add font pairing UI

#### Changes proposed in this Pull Request

* We copied the StylePreview component into Stepper from Gutenboarding.
* We copied effective font paring code from Gutenboarding. There were some TODOs about font loading from 2 years ago that we may want to address later.

#### Testing instructions

* http://calypso.localhost:3000/setup/fontPairing?flags=signup/anchor-fm&siteSlug=vlorran.wordpress.com&anchor_podcast=4a00d814
* Go to the above URL and click on the different fonts and make sure that the preview updates correctly.
* Click on the back button and select a new theme, then make sure that the font preview shows the new theme.

Related to #62666
